### PR TITLE
qrcp: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suyashbhawsar** [comment]
@kbdharun

File: pages/linux/qrcp.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# qrcp
+
+> A file transfer tool.
+> More information: <https://github.com/claudiodangelis/qrcp>.
+
+- Send a file(s) or directories:
```
Comment (CleanMachine1) [inline pages/linux/qrcp.md:6]:
```suggestion
- Send a file or directories:
```

File: pages/linux/qrcp.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`qrcp send {{path/to/file_or_directory}}`
```
Comment (CleanMachine1) [inline pages/linux/qrcp.md:8]:
```suggestion
`qrcp send {{path/to/file_or_directory path/to/file_directory ...}}`
```

File: pages/linux/qrcp.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Receive file(s):
```
Comment (CleanMachine1) [inline pages/linux/qrcp.md:10]:
```suggestion
- Receive files:
```

File: pages/linux/qrcp.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`qrcp receive`
+
+- [z]ip content before transferring:
+
+`qrcp send --zip {{path/to/file_or_directory}}`
+
+- Specify [p]ort to use for the server:
+
+`qrcp send/receive --port port`
```
Comment (CleanMachine1) [inline pages/linux/qrcp.md:20]:
```suggestion
`qrcp send/receive --port {{port_number}}`
```

File: pages/linux/qrcp.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Specify the network [i]nterface to use:
+
+`qrcp send/receive --interface interface`
+
+- Keep the server alive:
+
+`qrcp send/receive --keep-alive`
```
Comment (CleanMachine1) [inline pages/linux/qrcp.md:28]: this is incorrect command syntax, you cannot have commands which are actually 2 (the send/receive part)

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: pages/linux/qrcp.md (hunk 5 above)
Comment (suyashbhawsar) [reply pages/linux/qrcp.md:28]: > this is incorrect command syntax, you cannot have commands which are actually 2 (the send/receive part)

@CleanMachine1  So, what should I use there?

Comment (CleanMachine1) [reply pages/linux/qrcp.md:28]: Actually, for some reason, I forgot the syntax myself, we can just do the suggestions I have just added.

File: pages/linux/qrcp.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Send a file or directories:
+
+`qrcp send {{path/to/file_or_directory path/to/file_directory ...}}`
+
+- Receive files:
+
+`qrcp receive`
+
+- [z]ip content before transferring:
```
Comment (CleanMachine1) [inline pages/linux/qrcp.md:14]:
```suggestion
- Compress content before transferring:
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/qrcp.md (hunk 6 above)
Comment (suyashbhawsar) [reply pages/linux/qrcp.md]: It's using the zip archiving format, so wouldn't it be better to write zip?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**